### ISSUE TITLE
About train！

### ISSUE THREAD

**Issue by liutao23** [btxviny/DUT-Video-Stabilization#2]
Dear author, hello! Have you reproduced the training process of DUT? Could you please give me some guidance on how to organize the training dataset? I would be extremely grateful! I am unsure about how to train the DUT model for smooth kernel prediction and its grid point motion propagation module.

**Comment by btxviny** [comment]
No, I did not train the any of the sub models. This is just a refactored inference module, with a second option for the warping function.

**Comment by liutao23** [comment]
OK，I will try to try it，if I success，I will tell you，thank！
